### PR TITLE
removing unenrolled sites

### PR DESCRIPTION
**Pull request by apurva-s**
I have made the variables camel case, removed the errant console.log, ensured that setLoading is undisturbed and filtered on FarmDates and SensorVisuals using protocols_enrolled.

### CODE REVIEW

File: src/SensorVisuals/SensorVisuals.js [hunk 1]
```diff
@@ -146,7 +146,10 @@ const SensorVisuals = (props) => {
           }
         });
         setData(responseArray);
-        setLoading(false);
```
Comment (rbandooni) [inline src/SensorVisuals/SensorVisuals.js:149]: Why has this function been removed?

File: src/SensorVisuals/SensorVisuals.js [hunk 2]
```diff
@@ -146,7 +146,10 @@ const SensorVisuals = (props) => {
...
+        let response_with_filter = responseArray.filter((r) => {
```
Comment (mikahpinegar) [inline src/SensorVisuals/SensorVisuals.js:149]: should be camel cased

**Review by rbandooni — CHANGES_REQUESTED** [review]
Please fix the edits below

**Review by mikahpinegar — DISMISSED** [review]

File: src/SensorVisuals/SensorVisuals.js [hunk 3]
```diff
@@ -145,8 +145,12 @@ const SensorVisuals = (props) => {
             }
           }
         });
-        setData(responseArray);
         setLoading(false);
+        console.log(responseArray);
```
Comment (mikahpinegar) [inline src/SensorVisuals/SensorVisuals.js:149]: remove print

Comment (apurva-s) [reply src/SensorVisuals/SensorVisuals.js]: Removed the console.log

**Review by rbandooni — APPROVED** [review]

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**